pkg/instancegroups/instancegroups.go

```diff
@@ -113,20 +127,24 @@ func (c *DeleteInstanceGroup) DeleteInstanceGroup(group *api.InstanceGroup) erro
 
 		glog.Infof("Deleting AutoScalingGroup %q", group.ObjectMeta.Name)
 
-		err = cig.Delete(c.Cloud)
+		err = cig.Delete(d.Cloud)
 		if err != nil {
 			return fmt.Errorf("error deleting cloud resources for InstanceGroup: %v", err)
 		}
 	}
 
-	err = c.Clientset.InstanceGroupsFor(c.Cluster).Delete(group.ObjectMeta.Name, nil)
-	if err != nil {
-		return err
+	if err = d.Clientset.InstanceGroupsFor(d.Cluster).Delete(group.ObjectMeta.Name, nil); err != nil {
+		return fmt.Errorf("unable to delete instance group: %q, %v", group.ObjectMeta.Name, err)
 	}
 
 	return nil
 }
 
+const (
+	KOPS_IG_PARENT = "kops.alpha.kubernetes.io/instancegroup/parent"
```

Comment:
We are adding the annotations on to the instance group so that we have the lineage or history of the instance group.  Mostly I am using these to help with restarts and failures.  If an IG has a child, and that child exists, we do not create the child/replacement instance groups.  I need to check to see if the child exists :)